Applications/Viqui/vqApplication.cxx

```diff
@@ -1112,6 +1116,21 @@ void vqApplication::generateReport()
     }
 }
 
+//-----------------------------------------------------------------------------
+void vqApplication::exportSvm()
+{
+  this->Core->updateIqrModel();
+
+  QString path =
+    vgFileDialog::getSaveFileName(
+      this, "Location of SVM export", QString(), "SVM File (*.svm);;");
```

Comment:
Please use "Save IQR model..." as the dialog title; this is mostly consistent with other dialog titles. ("Mostly" meaning here that other dialogs are only mostly consistent :stuck_out_tongue:.) Also, the file type description should probably be "IQR Model File".